File: 406/406_A/main.go
package main

import (
	"fmt"
)

func main() {
	var h1, m1, h2, m2 int
	fmt.Scan(&h1, &m1, &h2, &m2)

	total := h1 * 60 + m1
	total1 := h2 * 60 + m2

	diff := total - total1
	if diff < 0 {
		fmt.Println("No")
	} else {
		fmt.Println("Yes")
	}
}

File: 406/406_A/main.py
time = list(map(int, input().split()))
h = []
m = []

for i in range(len(time)):
  if i % 2 == 0:
    h.append(time[i])
  else:
    m.append(time[i])

# 時間を分単位にする
total = h[0] * 60 + m[0]
total1 = h[1] * 60 + m[1]

diff = total - total1
if diff < 0:
  print("No")
else:
  print('Yes')
    